python/tvm/ir/transform.py

```diff
@@ -80,9 +88,13 @@ def __init__(
         if not isinstance(disabled, (list, tuple)):
             raise TypeError("disabled_pass is expected to be the type of " + "list/tuple/set.")
 
+        instruments = list(instruments) if instruments else []
+        if not isinstance(instruments, (list, tuple)):
+            raise TypeError("disabled_pass is expected to be the type of " + "list/tuple/set.")
```

Comment:
Add which type it actually is to the error message.

Reply:
Done

Reply:
also correct error message from `disabled_pass` -> `instruments`